csrankings.csv

```diff
@@ -1436,6 +1437,7 @@ Bin Ma,University of Waterloo,https://cs.uwaterloo.ca/~binma/,f-eL6DYAAAAJ
 Bin Ren,College of William and Mary,http://www.wm.edu/as/computerscience/faculty/ren_bin.php/,XU7ZT5kAAAAJ
 Bin Sheng,Shanghai Jiao Tong University,http://english.seiee.sjtu.edu.cn/english/detail/841_963.htm,unORJQ8AAAAJ
 Bin Sun,Peking University,http://icl.pku.edu.cn/members/bswen/index.html,NOSCHOLARPAGE
+Bin Wang 0004,Chinese Academy of Sciences,https://scholar.google.com/citations?user=tDajnHEAAAAJ&hl=en,tDajnHEAAAAJ

```

Comment:
This is not an appropriate home page. Please include an institutional home page or similar.